numpy/_core/src/umath/umathmodule.c

```diff
@@ -321,5 +322,9 @@ int initumath(PyObject *m)
         return -1;
     }
 
+    if (init_argparse_mutex() < 0) {
+        return -1;
+    }
+
```

Comment:
```suggestion
```
Umath is the same module now, we shouldn't need this here?!

Reply:
This is where this is initialized for the multiarray module. The umath initialization function is called inside the multiarray initialization.

`init_argparse_mutex` is called twice in this PR because the multiarray tests module is a different module, so I need to do the static initialization in that module too.

Reply:
Ah, sorry my bad, noticed that second time, but somehow thought I already saw it ini multiarray.c.